cmd/cluster/context.go

```diff
@@ -102,20 +106,45 @@ func (o *statusOptions) run() error {
 	defer connection.Close()
 
 	// Get limited support reasons for a cluster
-	collection := connection.ClustersMgmt().V1().Clusters()
-	resource := collection.Cluster(o.clusterID).LimitedSupportReasons()
-	response, err := resource.List().SendContext(ctx)
+	lsResponse, err := getLimitedSupportReasons(connection, ctx, o.clusterID)
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "Can't retrieve cluster limited support reasons: %v\n", err)
 		os.Exit(1)
 	}
 
 	// Check support status of cluster
-	checkSupportStatus(response)
+	checkSupportStatus(lsResponse)
```

Comment:
```suggestion
	printSupportStatus(lsResponse)
```
I think the current function name is a bit misleading, it doesn't return anything but get and print the supportstatus